src/WallGo/mathematicaHelpers.py

```diff
@@ -1,22 +1,70 @@
-import subprocess
 import pathlib
+import subprocess
+import sys
+
+# Put common Wolfram Mathematica and WallGoMatrix related functions here.
+# Common physics/math functions should go into helpers.py
+
+
+def generateMatrixElementsViaSubprocess(
+    inFilePath: pathlib.Path, outFilePath: pathlib.Path, bVerbose: bool = False
+) -> None:
+    """
+    Generates matrix elements by executing a Mathematica script via a subprocess.
+
+    This function takes the input and output file paths, converts them to string representations,
+    and constructs a command to run a Mathematica script using `wolframscript`. The command is
+    executed using the `subprocess.run` method, and the output is printed to the console. If the
+    command fails, an error message is printed.
+
+    This requires an active WolframKernel.
```

Comment:
What does "active WolframKernel" mean? I'm under the impression that wolframscript starts a new kernel when invoked.

Reply:
I think it's meant that it has to be "activated" (I got an error that mine wasn't)

Reply:
Isn't that the same thing?

Reply:
Active kernel sounds like it needs to be running prior to calling this function. IMO the description should say that you need to have a licensed installation of WolframEngine.